README.md

```diff
@@ -23,16 +22,27 @@ $ export EXPLORER=zzz
 
 4. **OPTIONAL** TLS or Onion
 
-#### TLS
+### TLS
+
+#### Trade interface
 
-Uncomment in the compose file (`docker-compose.yml`) the TLS related stuff and export ENV with the asbolute path to the SSL Certificate and Key to be used.
+To enable TLS encryption on the Trade interface of the daemon, you have to generate your own TLS key and certificate files. Once this is done, export the ENV variables with the absolute path of the files: 
 
 ```sh
-$ export SSL_CERT_PATH=/path/to/fullchain.pem
-$ export SSL_KEY_PATH=/path/to/privatekey.pem
+$ export SSL_CERT_PATH=/path/to/certificate.crt
```

Comment:
keep fullchain.pem, better safe than sorry (the name of lets ecnrypt certbot give by default)